pkg/settings/network_setting.go

```diff
@@ -0,0 +1,25 @@
+package settings
+
+import (
+	"encoding/json"
+	"github.com/sirupsen/logrus"
+)
+
+type NetworkSettings struct {
+	// physical NIC(network interface card)
+	NIC string
+
+	// previous configured NIC
+	PreviousNIC string
+}
+
+func initNetworkSettings() string {
```

Comment:
Can we do this on the network controller side? If there's nothing to do with the setting here, we don't need to define it here as well.

Reply:
updated.